Guides/Keyed.md

```diff
@@ -0,0 +1,78 @@
+# Indexed
+
+[[Source](https://github.com/apple/swift-algorithms/blob/main/Sources/Algorithms/Keyed.swift) | 
+ [Tests](https://github.com/apple/swift-algorithms/blob/main/Tests/SwiftAlgorithmsTests/KeyedTests.swift)]
+
+Stores the elements of a sequence as the values of a Dictionary, keyed by the result of the given closure.
+
+```swift
+let fruits = try! ["Apple", "Banana", "Cherry"]
+let fruitByLetter = fruits.keyed(by: { $0.first! })
+// Results in:
+// [
+//     "A": "Apple",
+//     "B": "Banana",
+//     "C": "Cherry",
+// ]
+```
+
+Duplicate keys throw an runtime error by default. Alternatively, you can provide a closure which specifies which value to keep:
```

Comment:
Grammar nitpick: "an runtime" should be "a runtime".

Reply:
Not a nitpick at all!